<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="start_btn">
    <property name="geometry">
     <rect>
      <x>836</x>
      <y>447</y>
      <width>170</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>170</width>
      <height>80</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;等线&quot;;
border：none;</string>
    </property>
    <property name="text">
     <string>开始游戏</string>
    </property>
   </widget>
   <widget class="QComboBox" name="size_comboBox">
    <property name="geometry">
     <rect>
      <x>836</x>
      <y>335</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>170</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;等线&quot;;
border：none;</string>
    </property>
    <property name="currentText">
     <string>中</string>
    </property>
    <item>
     <property name="text">
      <string>迷你</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>小</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>中</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>大</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="speed_comboBox">
    <property name="geometry">
     <rect>
      <x>836</x>
      <y>268</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>170</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;等线&quot;;
border：none;</string>
    </property>
    <property name="currentText">
     <string>3</string>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>270</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;等线&quot;;</string>
    </property>
    <property name="text">
     <string>速度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>330</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;等线&quot;;</string>
    </property>
    <property name="text">
     <string>尺寸：</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
